engine/src/main/java/org/hibernate/search/bridge/builtin/TikaBridge.java

```diff
@@ -182,6 +209,13 @@ private FileInputStream openInputStream(File file) {
 		}
 	}
 
+	private static class AutoDetectParserProvider implements TikaParserProvider {
```

Comment:
Couldn't we have only one instance of this one? TikaBridge could be used several times.

Wondering also if we could have only one instance of the parser for all the TikaBridges (IIRC the metadata are read sequentially so we shouldn't have concurrent accesses?).

Reply:
> Not sure it's obvious, as the AutoDetectParser relies on system properties and environment variables for its configuration, and those can change. We can create a ticket if you want, but I wouldn't try that without serious regression testing.

Yeah, OK, considering I would like to consider backporting it, let's be conservative :).